app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="categories_sect">
+<div class="navigation">
+  <span class="hamburger">&#9776;</span>
+   <h1>Categories</h1>
+  <%= link_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out-butt" %>
+</div>   
+    <div class="cat_page">
+  <div id="categories">
+    <% if @categories.length == 0 %>
+      <p class="no-categories"> There are no categories yet,,,,!create one</p>
+    <% else %>
+      <% @categories.each do |category| %>
```

Comment:
- [ ] Kindly increase the top, left, and right spacing for each category item to match the design guidelines.